# ele vai achar porque a função está num arquivo do mesmo diretório
from potencia import pow

print(pow(2, 3))
# resultado é 8

print(pow(2))
# resultado é 4, pois não foi passado valor para b, então ele, usa o default definido na função (2)"